src/image_function.cpp

```diff
@@ -58,19 +58,23 @@ namespace
         Image_Function::ParameterValidation( image, x, y, width, height );
         Image_Function::VerifyGrayScaleImage( image );
 
+	 if (dilationX == 0u && dilationY == 0u) {
```

Comment:
Please do not use `{` and `}` for single line code inside if block to make code look nicer :)